app/controllers/placements/pages_controller.rb

```diff
@@ -1,3 +1,9 @@
 class Placements::PagesController < ApplicationController
   skip_before_action :authenticate_user!
+
+  def start
```

Comment:
Could you call this `landing_page` to keep it consistent with the name of the Markdown file?

```suggestion
  def landing_page
```

Reply:
I called it start to keep it consistent with FM. Happy to change it though @ollietreend